openqabot/pc_helper.py

```diff
@@ -20,11 +21,7 @@ def fetch_matching_link(url, regex):
         # So, the first link matching the regex is the most recent one
         req = requests.get(url + "/?C=M;O=A")
         text = req.text
-    except (
-        ConnectionError,
-        Timeout,
-        ValueError,
-    ) as err:
+    except (ConnectionError, Timeout, ValueError, HTTPError) as err:
```

Comment:
I would actually go further and just catch parent error class to mute ANY errors . Enumerating errors make sense only when situation is "Ok I know that errors A, B and C may happen let's catch them but anything else happens I would like throw it further " and it is NOT the case here . Here we saying "OK whatever error happened I want to mute it " . Because some day in future we will be here again adding yet-another-error-class into the list

Reply:
hmm, also possibility `BaseException` should work